infra/terraform/prod/cloudfront.tf

```diff
@@ -1,46 +1,96 @@
-resource "aws_cloudfront_distribution" "wikijump_cf_distro" {
-    enabled                 = true
-    is_ipv6_enabled         = true
-    default_root_object     = "index.php"
-
-    aliases                 = [var.web_domain, var.files_domain]
-
-    origin {
-        domain_name         = aws_lb.wikijump_elb.dns_name
-        origin_id           = "wikijump_elb"
-        custom_header {
-            name            = "X-CLOUDFRONT-WIKIJUMP-AUTH"
-            value           = var.cf_auth_token
-        }
-    }
-
-    restrictions {
-      geo_restriction {
-        restriction_type  = "none"
-      }
-    }
-
-    default_cache_behavior {
-    allowed_methods         = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
-    cached_methods          = ["GET", "HEAD"]
-    target_origin_id        = "wikijump_elb"
-
-    forwarded_values {
-      query_string          = true
-
-      cookies {
-        forward             = "all"
-      }
-    }
-
-    viewer_protocol_policy  = "redirect-to-https"
-    min_ttl                 = 0
-    default_ttl             = 60
-    compress                = true
-    max_ttl                 = 60
-    }
-
-    viewer_certificate {
-        acm_certificate_arn = aws_acm_certificate.cf_wildcard_cert.arn
-  }
-}
\ No newline at end of file
+# resource "aws_cloudfront_distribution" "wikijump_cf_distro" {
```

Comment:
Commented code. 

If we're just disabling this file, maybe rename it? `cloudfront.tf -> cloudfront.disabled_tf`

Reply:
I didn't mean to touch prod stuff in this, I'm not going to worry about it just at the moment since almost all of it's going to change eventually. Blame VS code for any files in prod that were touched.